tweepy/parsers.py

```diff
@@ -54,11 +54,11 @@ def parse(self, method, payload):
             raise TweepError('Failed to parse JSON payload: %s' % e)
 
         needs_cursors = 'cursor' in method.session.params
-        if needs_cursors and isinstance(json, dict):
-            if 'previous_cursor' in json:
-                if 'next_cursor' in json:
-                    cursors = json['previous_cursor'], json['next_cursor']
-                    return json, cursors
+        if needs_cursors and isinstance(json, dict) \
+                and 'previous_cursor' in json \
+                and 'next_cursor' in json:
+            cursors = json['previous_cursor'], json['next_cursor']
+            return json, cursors
```

Comment:
This will still not return any cursors if only `next_cursor` is present in the response. Looks like it should return `json, (None, next_cursor)` if there's only one?